### PR TITLE
I18next update

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

**Comment by damencho** [comment]
This PR depends on updating all languages in pootle from using params as __app__ to {{app}}, which I'm currently working on in parallel of adding some new languages and updating others.

File: react/features/base/i18n/functions.js [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
 /**
  * Wraps a specific React Component in order to enable translations in it.
  *
  * @param {Component} component - The React Component to wrap.
- * @param {Object} options - Additional options to pass into react-i18next's
- * initialization.
  * @returns {Component} The React Component which wraps {@link component} and
  * enables translations in it.
  */
-export function translate(component, options = { wait: true }) {
```
Comment (virtuacoplenny) [inline react/features/base/i18n/functions.js]: wait is no longer needed? I see ChatMessage.js was using this options object.

Comment (virtuacoplenny) [reply react/features/base/i18n/functions.js]: Edit: nevermind, I see you looked into it below.

**Comment by damencho** [comment]
This update give us one more thing, we can un-wrap all components from this translate(component) and move away from storing t in props. 
https://react.i18next.com/latest/usetranslation-hook 
https://react.i18next.com/latest/trans-component

File: lang/main.json [hunk 2]
```diff
@@ -49,7 +45,7 @@
         "today": "Today"
     },
     "chat": {
-        "error": "Error: your message \"__originalText__\" was not sent. Reason: __error__",
+        "error": "Error: your message \"{{originalText}}\" was not sent. Reason: {{error}}",
```
Comment (saghul) [inline lang/main.json]: side comment: I think putting the entire message there is not nice. I'd just say "Your message could not be sent".

File: react/features/base/i18n/i18next.js [hunk 3]
```diff
@@ -40,22 +40,26 @@ const options = {
...
+    },
 
     // XXX i18next modifies the array lngWhitelist so make sure to clone
     // LANGUAGES.
-    lngWhitelist: LANGUAGES.slice(),
-    load: 'unspecific',
-    ns: {
-        defaultNs: 'main',
-        namespaces: [ 'main', 'languages', 'countries' ]
+    whitelist: LANGUAGES.slice(),
```
Comment (saghul) [inline react/features/base/i18n/i18next.js]: style: sort alphabetically

**Pull request merged**